index.html

```diff
@@ -21,15 +21,11 @@
         </div>
```

Comment:
- [ ] Make sure your page has the proper background image (decorations are always background images, not `img` tags). Example `background-image: url("./image/image1.png");` 🧱 _**If an image is not part of the content use CSS background-image**_

![image](https://user-images.githubusercontent.com/43172164/200026404-ce7cf545-4666-4469-be77-1b42e41044d4.png)